oxidation/libparsec/crates/platform_device_loader/src/native/local_device_file.rs

```diff
@@ -267,6 +278,6 @@ pub fn change_device_password(
     old_password: &str,
     new_password: &str,
 ) -> Result<(), LocalDeviceError> {
-    let device = LocalDevice::load_device_with_password(key_file, old_password)?;
```

Comment:
Why did you moved this function? It doesn't make sense to be part of `LocalDevice`?

Reply:
This method is `platform-specific` so if I understand what @touilleMan wants, this should be part of `platform_device_loader`

Reply:
Ok, this will conflict with #3896 then